docs/engines/01_available_engines.rst

```diff
@@ -0,0 +1,124 @@
+
+Available Engines
+=================
+
+
+
+The tables below give a high-level overview of the planning engines integrated with the UP.
+
+Action-Based Planning
+^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: 
+
+  * - Engine
+    - Operation modes
+    - Classical
+    - Numeric
+    - Temporal
+    - Metrics
+  * - `Fast-Downward`_
+    - OneShot, Anytime
+    - 🗸
+    - 
+    - 
+    - plan length, action costs
+  * - `ENHSP`_
+    - OneShot, Anytime
+    - 🗸
+    - 🗸
+    -
+    - action costs, final value
+  * - `Tamer`_
+    - OneShot
+    - 🗸
+    - 🗸
+    - 🗸
+    - 
+  * - `Aries`_ [#aries-actions]_
+    - OneShot, Anytime
+    - 🗸
+    - 🗸 (integers)
+    - 🗸
+    - plan length, makespan, action costs
+  * - `Pyperplan`_ [#pyperplan-note]_
+    - OneShot
+    - 🗸
+    - 
+    - 
+    - 
+
+.. [#aries-actions] Aries' focus is on hierarchical planning and scheduling and is likely not competitive with other planners in action-based planning.
+.. [#pyperplan-note] Pyperplan is mostly intended for education purposes and cannot be expected to scale to non-trivial problems.
+
+
+Plan Validation
+^^^^^^^^^^^^^^^
+
+.. list-table::
+
+  * - Engine
+    - Action-Based planning
+    - Numeric
+    - Temporal
+    - Hierarchical
+    - Scheduling
+  * - `UP (builtin)`
+    - 🗸
+    - 
```

Comment:
UP plan validator supports numeric planning.

Reply:
Added. I will ask for input from engine developpers to complete it once this is merged.